client/src/components/Nav.js

```diff
@@ -15,6 +15,7 @@ import { ScrollLink, scrollSpy } from "react-scroll"
 import { bindActionCreators } from "redux"
 import Settings from "settings"
 import utils from "utils"
+import { PENDING_ASSESSMENTS_BY_POSITION } from "../pages/insights/Show"
```

Comment:
Components should as a general principle not import from pages (the other way around is fine).